src/test/kotlin/net/atos/zac/app/informatieobjecten/InformatieObjectenRESTServiceTest.kt

```diff
@@ -146,6 +152,62 @@ class InformatieObjectenRESTServiceTest : BehaviorSpec() {
                             enkelvoudigInformatieObjectData.beschrijving,
                             "geen"
                         )
+                        httpSession.removeAttribute("FILE_$documentReferentieId")
+                    }
+                }
+            }
+        }
+        given("an enkelvoudig informatieobject has been uploaded, and the zaak is open") {
+            When("createEnkelvoudigInformatieobject is called but the ZGW client service throws an exception") {
+                then("the enkelvoudig informatieobject is not to the zaak but is removed from the HTTP session") {
```

Comment:
Perhaps
```suggestion
                then("the enkelvoudig informatieobject is not in the zaak but is removed from the HTTP session") {
```

Reply:
meant to say 'not added to the zaak'. ;-) fixed it now. thanks!